### PR TITLE
type-checking rule for integer numbers

### PR DESCRIPTION
**Pull request by dpisarewski**
solves #405

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3252,6 +3253,23 @@ resource cleanup when possible.
...
   end
   ```
+## Numbers
+
+* <a name="integer-type-checking"></a>
+  Use `Integer` check type of an integer number. Since `FixNum` is platform-dependent, checking against it will
+  return different results on 32-bit and 64-bit machines.
+<sup>[[link](#integer-type-checking)]</sup>
+
+  ```
```
Comment (bbatsov) [inline README.md]: You forgot to add `ruby` here.

File: README.md [hunk 2]
```diff
@@ -3252,6 +3253,23 @@ resource cleanup when possible.
...
+  timestamp = Time.now.to_i
+  
+  #bad
+  timestamp.is_a? FixNum
```
Comment (bbatsov) [inline README.md]: It's Fixnum and Bignum.

File: README.md [hunk 3]
```diff
@@ -3252,6 +3253,23 @@ resource cleanup when possible.
...
+
+* <a name="integer-type-checking"></a>
+  Use `Integer` check type of an integer number. Since `FixNum` is platform-dependent, checking against it will
+  return different results on 32-bit and 64-bit machines.
+<sup>[[link](#integer-type-checking)]</sup>
+
+  ```
+  timestamp = Time.now.to_i
+  
+  #bad
```
Comment (bbatsov) [inline README.md]: Add a space between `#` and `bad`

**Comment by bbatsov** [comment]
Overall it looks good. I've added a few small remarks.

**Comment by dpisarewski** [comment]
Thanks @bbatsov. I corrected it according to your remarks.

**Pull request merged**